docs/misc/mo-algo-intro.md

```diff
@@ -1,5 +1,5 @@
 author: StudyingFather, Backl1ght, countercurrent-time, Ir1d, greyqz, MicDZ, ouuan
 
-莫队算法是由莫涛提出的算法。在莫涛提出莫队算法之前，莫队算法已经在 Codeforces 的高手圈里小范围流传，但是莫涛是第一个对莫队算法进行详细归纳总结的人。莫涛提出莫队算法时，只分析了普通莫队算法，但是经过 OIer 和 ACMer 的集体智慧改造，莫队有了多种扩展版本。
+Mo's algorithm is an algorithm proposed by [Mo Tao](https://www.zhihu.com/people/mythly) . Before him, the Mo's algorithm has been used in a small group on Codeforces, but Mo was the first person to ever summarize the algorithm in detail. When Mo proposed that algorithm, he only analyzed the ordinary version, but after the countless contributions from many OIers and ACMers, Mo's algorithm now has a variety of expanded versions.
 
-莫队算法可以解决一类离线区间询问问题，适用性极为广泛。同时将其加以扩展，便能轻松处理树上路径询问以及支持修改操作。
+The Mo's algorithm can solve a type of offline interval query problems and its application is extremely wide. At the same time, it can be easily expanded to handle queries on tree paths and support modification operations.
```

Comment:
```suggestion
The Mo's algorithm can solve a series of offline interval query problems and its applications are extremely wide. At the same time, it can be easily expanded to handle queries on tree paths and support modification operations.
```